setup.py

```diff
@@ -24,9 +24,9 @@
 setuptools.setup(
     name=fairlearn.__name__,
     version=fairlearn.__version__,
-    author="Miroslav Dudik, Richard Edgar, Brandon Horn, Roman Lutz",
-    author_email="fairlearn@microsoft.com",
-    description="Algorithms for mitigating unfairness in supervised machine learning",
+    author="Miroslav Dudik, Richard Edgar, Adrin Jalali, Michael Madaio, Roman Lutz, Hilde Weerts",
```

Comment:
```suggestion
    author="Miroslav Dudik, Richard Edgar, Adrin Jalali, Roman Lutz, Michael Madaio, Hilde Weerts",
```
Assuming the intention was to sort on last name, believe the L comes before the M in our alphabet :D

Reply:
I'm glad people read the stuff I write 🤣 thanks, @hildeweerts